packages/search-adapter/src/empathy/mappers/request/params/empathy-request-filters-solr-syntax.mapper.ts

```diff
@@ -72,7 +74,18 @@ export class EmpathyRequestFiltersSolrSyntaxMapper implements Mapper<Filter[], s
   }
 
   /**
-   * Check if the filters passed are of type {@link @empathyco/x-types#EditableNumberRangeFilter | EditableNumberRangeFilter}.
+   * Check if the filters passed are of type {@link @empathyco/x-types-next#RawFilter | RawFilter}.
+   *
+   * @param filters - The array of filters to check.
+   *
+   * @internal
+   */
+  private isArrayOfRawFilters(filters: Filter[]): filters is RawFilter[] {
+    return isRawFilter(filters[0]);
```

Comment:
I'm not sure about this method name, what do you want to check? I mean, the description says one thing but the name another.

Reply:
I guess it's just type narrowing.